mypy/test/testmerge.py

```diff
@@ -172,10 +171,7 @@ def format_symbol_table_node(self, node: SymbolTableNode) -> str:
             if node.kind == UNBOUND_IMPORTED:
                 return "UNBOUND_IMPORTED"
             return "None"
-        if isinstance(node.node, Node):
-            s = f"{str(type(node.node).__name__)}<{self.id_mapper.id(node.node)}>"
-        else:
-            s = f"? ({type(node.node)})"
```

Comment:
I was not sure if this is a guard against weird plugins or just unreachable code.

Reply:
Looks like mypy-primer does not report on any plugins that we check.